examples/Ensemble/requirements.txt

```diff
@@ -0,0 +1 @@
+seaborn
```

Comment:
Rename `h1st_Ensemble_Tutorial.ipynb` to `ensemble_tutorial.ipynb`.

Also we should reconsider mixing `examples` and `tutorial` in the same project subdir, cc @aht .
Perhaps tutorials need their own repo, @nqbao @aht @thangtp .